### PR TITLE
Wdio cucumber jasmine

### PR DESCRIPTION
**Pull request by wswebcreation**
Add Cucumber and Jasmine examples

### CODE REVIEW

File: wdio-cucumber/package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "SauceLabsVisualExampleforWebdriverIOCucumber",
```
Comment (FriggaHel) [inline wdio-cucumber/package.json:2]: This is the correct format (even if others don't respect it).
```suggestion
  "name": "saucelabs-visual-example-webdriverio-cucumber",
```

File: wdio-jasmine/package.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "SauceLabsVisualExampleforWebdriverIOJasmine",
```
Comment (FriggaHel) [inline wdio-jasmine/package.json:2]: Same as previous:
```suggestion
  "name": "saucelabs-visual-example-webdriverio-jasmine",
```

**Review by FriggaHel — COMMENTED** [review]
Overall LGTM.

Could you also add them to the CI ?
https://github.com/saucelabs/visual-examples/blob/main/.github/workflows/test.yml

**Comment by wswebcreation** [comment]
> Overall LGTM.
> 
> Could you also add them to the CI ? https://github.com/saucelabs/visual-examples/blob/main/.github/workflows/test.yml

@FriggaHel 

Done

File: wdio/README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Getting started with WebdriverIO [![](https://badgen.net/badge/Run%20this%20/README/5B3ADF?icon=https://runme.dev/img/logo.svg)](https://runme.dev/api/runme?repository=git%40github.com%3Asaucelabs%2Fvisual-examples.git)
+# Getting started with WebdriverIO Mocha [![](https://badgen.net/badge/Run%20this%20/README/5B3ADF?icon=https://runme.dev/img/logo.svg)](https://runme.dev/api/runme?repository=git%40github.com%3Asaucelabs%2Fvisual-examples.git)
```
Comment (FriggaHel) [inline wdio/README.md:1]: To stay consistent with other `README.md`:
```suggestion
# Getting started with WebdriverIO + Mocha [![](https://badgen.net/badge/Run%20this%20/README/5B3ADF?icon=https://runme.dev/img/logo.svg)](https://runme.dev/api/runme?repository=git%40github.com%3Asaucelabs%2Fvisual-examples.git)
```

**Review by FriggaHel — APPROVED** [review]

**Pull request merged**